### ISSUE TITLE
Error when using with next js 

### ISSUE THREAD

**Issue by aakash19here** [upstash/rag-chat#71]
./node_modules/.pnpm/undici@6.19.8/node_modules/undici/lib/web/fetch/util.js
Module parse failed: Unexpected token (860:57)
|       // 5. If object is not a default iterator object for interface,
|       //    then throw a TypeError.
>       if (typeof this !== 'object' || this === null || !(#target in this)) {
|         throw new TypeError(
|           `'next' called on an object that does not implement interface ${name} Iterator.`

**Comment by aakash19here** [comment]
```ts
import { RAGChat, openai } from '@upstash/rag-chat';
import { initVectorStoreClient } from './vector-store';

export const ragChat = new RAGChat({
  model: openai('gpt-4o'),
  vector: initVectorStoreClient(),
  namespace: 'udel',
});
```

Route handler 
```tsx
import { ragChat } from "@/utils/rag-chat";
import { NextResponse } from "next/server";

export const POST = async (req: Request) => {
  const { message } = await req.json();
  const { output } = await ragChat.chat(message);
  return NextResponse.json({ output });
};


```

**Comment by ytkimirti** [comment]
Hey @aakash19here, the issue is probably from nextjs. You can check this issue https://github.com/elastic/elasticsearch-js/issues/2207. 

Can you try bumping your nextjs version?

**Comment by aakash19here** [comment]
Thanks for the response @ytkimirti 

I did bump my next version and now I'm getting a tiktoken error 

<img width="1427" alt="Screenshot 2024-09-17 at 1 20 47 PM" src="https://github.com/user-attachments/assets/398b3dd9-cce8-4c10-97fc-7c16ef5a9dd4">

**Comment by CahidArda** [comment]
Hi @aakash19here,

We released version 2.0.0 and this error shouldn't occur. Can you try again?

**Comment by CahidArda** [comment]
closing due to inactivity.
